readme.md

```diff
@@ -174,83 +192,101 @@ app.get('/users', (req, res) => {
     res.json(users);
 });
 ```
-- After adding new routes and restarting the server, visit localhost:4000/users or localhost:4000/tasks from your browser! Or simply,
-in your console type:
+
+After adding new routes and restarting the server, visit localhost:4000/users or localhost:4000/tasks from your browser! Or in your console, simply type:
+
 ```
 $ curl localhost:4000/users
+```
+
+```
 $ curl localhost:4000/tasks
 ```
-- Let's imagine we would like to pull users details providing name, we can do it by passing in params, let's assume we know the first names of our users and this is how we would like to find them. Add the following GET method to our index.ts:
+
+Let's imagine we would like to get a specific user by their name. We can do this by passing in parameters. Let's assume we know the first names of our users and this is how we would like to find them. Add the following GET method to our `index.js`:
+
 ```js
 app.get('/users/:userName', (req, res) => {
     res.send(users[users.findIndex(u => u.firstName === req.params.userName)]);
   });
 ```
-and give it a try by using curl or visiting localhost:4000/users/Michael.
+
+and give it a try by using curl or visiting `localhost:4000/users/Michael`.
+
 ```
 $ curl localhost:4000/users/Michael
 ```
-In your response you should have got details concerning Michael only, however, as this GET request takes in PARAM, you can just change the URL to any other name that is in our database to pull the record of the required user! Try this in your command line:
+
+In your response you should have received details concerning Michael only. As this GET request takes in a parameter, you can just change the URL to any other name that is in our database to pull the record of the required user! Try this in your command line:
+
 ```
 $ curl localhost:4000/users/John
 ```
+
 See how cool REST can be?! Now, let's go through another HTTP method - DELETE! To do that add another method to our server:
+
 ```js
-app.delete('/users/delete/:userName', (req, res) => {
+app.delete('/users/:userName', (req, res) => {
```

Comment:
With REST, you can have the exact same URL for GET, POST, DELETE, PUT etc. It is the HTTP protocol that dictates which endpoint your request hits.